paz/backend/boxes.py

```diff
@@ -5,60 +5,47 @@
 
 
 def split(boxes):
+    """Split boxes into x_min, y_min, x_max, y_max components."""
     return jp.split(boxes, 4, axis=1)
 
 
-def join(coordinate_0, coordinate_1, coordinate_2, coordinate_3):
-    return jp.concatenate(
-        [coordinate_0, coordinate_1, coordinate_2, coordinate_3], axis=1
-    )
-
-
-def to_center_form(boxes):
-    """Transform from corner coordinates to center coordinates.
-
-    # Arguments
-        boxes: Numpy array with shape `(num_boxes, 4)`.
-
-    # Returns
-        Numpy array with shape `(num_boxes, 4)`.
-    """
-    x_min, y_min, x_max, y_max = split(boxes)
+def compute_centers(x_min, x_max, y_min, y_max):
```

Comment:
I think this function should take boxes as input, not the coordinates given that we will be calling:
`paz.boxes.computer_center(boxes)`